### PR TITLE
Use language parameter to create exports

### LINKED ISSUE #2017: Use query parameter passed to exports endpoint to create exports with language other than the default
## Suggested Feature / Enhancement

Use language query parameter passed to export builder [here](https://github.com/onaio/onadata/blob/master/onadata/libs/utils/export_tools.py#L191) to create exports in a language different from the default

### PR DESCRIPTION
**Pull request by FrankApiyo**
Closes #2017

### CODE REVIEW

File: onadata/apps/logger/models/xform.py [hunk 1]
```diff
@@ -534,13 +534,16 @@ def get_language(self, languages, language_index=0):
...
+    def get_label(self, abbreviated_xpath, elem=None, language=None):
         elem = self.get_element(abbreviated_xpath) if elem is None else elem
 
         if elem:
             label = elem.label
 
-            if isinstance(label, dict):
+            if language and isinstance(label, dict):
+                label = label[language]
+
```
Comment (DavisRayM) [inline onadata/apps/logger/models/xform.py:545]: Should we handle cases where a user passes a language that doesn't exist on the XForm to the `get_label` function ? _Seems the `self.get_language` function has a list of languages... or we could just use `label.get(language, <default>)`.._

Or is that something that's handled else where ?

Comment (FrankApiyo) [reply onadata/apps/logger/models/xform.py:545]: Such a situation has not been taken in to account; going to tweak this a bit

**Review by DavisRayM — APPROVED** [review]
LGTM

**Review by ivermac — DISMISSED** [review]
looks good to me

**Review by ivermac — APPROVED** [review]

**Review by DavisRayM — APPROVED** [review]

**Review by ivermac — APPROVED** [review]
Looks good to me

**Pull request merged**